src/components/ArtistName.jsx

```diff
@@ -0,0 +1,18 @@
+import { PropTypes } from "prop-types";
+import "./ArtistName.css";
+
+export const ArtistName = ({ artists }) => {
+  return (
+    <div className="artist-container">
+      {artists.map((artist) => (
+        <span key={artist.id} className="artist-name-link">
+          <a href={artist.external_urls.spotify}>{artist.name} </a>
```

Comment:
there's an extra space between `{artist.name}` and `</a>` on line 9. It's causing the artist name to be rendered somewhat incorrectly if there are 2 or more artists. Can we remove that space?


![Screenshot 2023-10-21 at 20 47 27](https://github.com/themisterkai/project-music-releases-vite/assets/9663970/73361326-2552-4858-9e8f-99f10a87ecc6)

Reply:
Of course, I'll remove it